restos.json

```diff
@@ -2,5 +2,6 @@
   "lyon":[
     {"name":"L'Absteme", "url":"http://www.labsteme.fr/en/", "type":"😜"},
     {"name":"Stamtich", "url":"https://www.facebook.com/stamtich/", "type":"Alsacien"}
+    {"name":"Les Raffineuses", "url":"41 Rue Professeur Grignard, 69007 Lyon", "type":"benbon"}
```

Comment:
you're right. I was looking to a proper rendering of the 2 lines info, but you're the boss. 🤖